rpc/amqp-rpc.go

```diff
@@ -95,6 +97,16 @@ func AMQPDeclareExchange(conn *amqp.Connection) error {
 	return err
 }
 
+// Create a quick consumer ID we can recreate later if we need to cancel
+// a consumer.
+func getConsumerName(serverQueue string) (string, error) {
+	hostname, err := os.Hostname()
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%s.%s", serverQueue, hostname), nil
```

Comment:
This is not guaranteed to be unique during local development, and so seems likely to cause bugs, no?

Could you say more about why we need this name?